docs/development.md

```diff
@@ -85,8 +85,8 @@ non-generated code in the commit history.
 
 ## Running E2E Tests
 
-The KubeFed E2E tests must be executed against a deployed
-federation of one or more clusters.  Optionally, the KubeFed
+The KubeFed E2E tests must be executed against a KubeFed control plane
+with one or more registered clusters.  Optionally, the KubeFed
```

Comment:
Just a note, I'm personally ok with the use of 'register' here and think it reads in a way that makes sense. If people feel strongly about it later we can always change it back.